### PR TITLE
WAR-1124: Changed bitbucket to github in katana topbar

### PR DESCRIPTION
**Pull request by sanikakulkarni**

### CODE REVIEW

File: katana/views/index.html [hunk 1]
```diff
@@ -54,7 +54,7 @@
             </a>
             <a target="_blank" href="https://github.com/warriorframework/warriorframework">
                 <div class="pull-right big-navbar-icons tooltip-on-right-dark">
-                    <i class="fa fa-bitbucket fa-2x" aria-hidden="true"></i>
+                    <i class="fa fa-github fa-2x" aria-hidden="true"></i>
                     <span class="tooltip-content">Warrior Repository</span>
                 </div>
                 <div class="pull-right small-navbar-icons tooltip-on-right-dark">
```
Comment (kaheichan) [inline katana/views/index.html]: What is the line below for? it mentions bitbucket, also L32 has something we don't want

Comment (sanikakulkarni) [inline katana/views/index.html]: That bitbucket has been changed to github

**Comment by sanikakulkarni** [comment]
Updated the pull request, so that the icons are accessible right at the landing page.

**Comment by kaheichan** [comment]
@sanikakulkarni 
Check this line out
https://github.com/warriorframework/warriorframework/pull/134/files#diff-ba263fb4e4d8923b3ad7cbaed64d211aR28

**Pull request merged**